tests/test_environment_LocalDescriptors.py

```diff
@@ -239,11 +239,12 @@ def test_ql(self, unit_cell):
             # in cases where there is no symmetry. Since simple cubic
             # should have a 0 ql value in many cases, we need to set high
             # tolerances for those specific cases.
-            atol = 1e-3 if unit_cell == freud.data.UnitCell.sc else 1e-6
+            atol = 1e-3 if unit_cell == freud.data.UnitCell.sc else 1e-5
```

Comment:
This seems unrelated.

Reply:
The changes on this PR created some loss of precision in the `LocalDescriptors` code. I'm not sure what the origin is.

Reply:
Hmm well if it is only sc I guess that is fine. I didn't notice anything in the code myself.